platform/lib/routers/growPages.js

```diff
@@ -178,6 +188,45 @@ const optimizer = AmpOptimizer.create({
   ],
 });
 
+// Matches component ref doc URLs. Redirects latest version to non-versioned URL
+growPages.get(
+    /^(\/[a-z]+)?\/documentation\/components\/(amp-[a-z0-9-]+)(-v(\d\.\d))?\//,
+    async (req, res, next) => {
+      const component = req.params[1];
+      let version = req.params[3];
+      const versionsByFormat = formatComponentMapping[component];
+      if (!versionsByFormat) {
+        signale.warn(`No version mapping defined for ${component}. Run 'gulp importAll' to fix.`);
+        return;
+      }
+      const format = getFormatFromRequest(req);
+
+      // add format supported versions to template context
+      let versions = versionsByFormat[format];
+      if (!versions) {
+        signale.warn(`No version mapping defined for format ${format}. Try running 'gulp ` +
+        'importAll\' to fix.');
+        // set to the latest available version and tell the user on the page
+        versions = [versionsByFormat.current];
+      }
+      const latestVersionByFormat = versions[versions.length - 1];
+
+      // redirect to unversioned URL for latest format specific version
+      if (version >= latestVersionByFormat) {
```

Comment:
String comparison is wrong here (2.0 vs 11.0)

Reply:
done